Command.fs

```diff
@@ -21,12 +21,24 @@ open Starling.Core.Traversal
 [<AutoOpen>]
 module Types =
     /// <summary>
-    ///     A command.
-    ///
-    ///     <para>
-    ///         A command is a list, representing a sequential composition
-    ///         of primitives represented as <c>PrimCommand</c>s.
-    ///     </para>
+    ///     Mid-level register transfer logic used to encode Starling
+    ///     commands.
+    /// </summary>
+    /// <typeparam name="L">The type of lvalues.</typeparam>
+    /// <typeparam name="RV">The type of rvalue variables.</typeparam>
+    type Microcode<'L, 'RV> when 'RV : equality =
+        /// <summary>An assignment, perhaps nondeterministic.</summary>
+        | Assign of lvalue : 'L * rvalue : Expr<'RV> option
```

Comment:
Most of this has just moved from `Model`, except this line.  This has grown an `option`, which encodes havoc when `None`.